4MD/src/main/java/com/infoshareacademy/four_md/model/User.java

```diff
@@ -1,26 +1,23 @@
 package com.infoshareacademy.four_md.model;
+import java.util.List;
 
 public class User {
-    private int userId;
+
+    private static int idCache = 0;
```

Comment:
Wydaje mi się, że nie jesteś odpowiedzialny za przypisywanie id-ków.
(To jest problem Michała)